releases/handbook.md

```diff
@@ -194,15 +193,38 @@ Criteria for timeline that the team needs to consider
 - Kubecon dates - let’s not hard block on events, but keep them in mind since we know community members might get doublebooked.
 - Associated events (aka. AI Day at Kubecon, Tensorflow events) - we want to keep them in mind.
 
-**Success Criteria:** Release team selected, schedule sent to kubeflow-discuss, all release team members have the proper permissions and are meeting regularly.
+**Success Criteria:**
+* Release team selected
+* Schedule sent to kubeflow-discuss
+* All release team members have the proper permissions and are meeting regularly
```

Comment:
I think it would be helpful to list where the new release team members need to be added for permission. For example, the release GitHub team needs to be updated https://github.com/kubeflow/internal-acls/pull/511 and you mentioned a new member need to join kubeflow-discuss mailing group. Can we add those here and is there anything else that we can specify here?

Reply:
I've mentioned that everyone needs to be part of the `internal-acl` repo, although it's a hyperlink, in the list of checkboxes for the [preparation phase](https://github.com/kubeflow/community/blob/master/releases/handbook.md#preparation).

I'll make it more explicit by adding sub-checks in that step to note that release team mambers need to be:
1. In the internal-acl repo and github team
2. Part of the `kubeflow-discuss` mailing list, in order to have access to the assets (drive, notes, recordings) of the release team

WDYT?

Reply:
again, totally missed the preparation phase. 

I think the only item missing is being part of the `kubeflow-discuss`. I think just adding that should be sufficient!